examples/tutorial/jupyter/execution/omnisci_on_native/jupyter_omnisci_env.yml

```
@@ -0,0 +1,6 @@
+name: jupyter_modin_on_omnisci
+channels:
+  - conda-forge
+dependencies:
+    - modin-omnisci
+    - jupyter
```

```suggestion
name: jupyter_modin_on_omnisci
channels:
  - conda-forge
dependencies:
  - modin-omnisci
  - jupyter
```
Please check the deps are installable from this file. There was incorrect indentation.

---

@YarShev This was changed